src/App.jsx

```diff
@@ -1,35 +1,59 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import * as React from "react";
+import TodoList from "./TodoList.jsx";
+import AddTodoForm from "./AddTodoForm.jsx";
+import { useState } from "react";
+import Quickmaths from "./Quickmaths.jsx";
```

Comment:
Hey Grant, it looks to me like you can remove this import altogether. I'm curious, what purpose does this Quickmaths module serve?

It's currently giving me errors when I try to run your code locally.

`[plugin:vite:import-analysis] Failed to resolve import "./Quickmaths.jsx" from "src/App.jsx". Does the file exist?`

<img width="1398" alt="image" src="https://github.com/user-attachments/assets/b32f3308-6b21-44f8-b2b0-ccb727ec8804">

Reply:
it was a file i created so i could test out doing props, originally i was trying to make a separate branch for testing out concepts outside of the required assignment, i just added it to the git-ignore, but if you could help me separate it into a different branch next week would be awesome. I added the file back though so it should no longer error